### ISSUE TITLE
examples

### ISSUE THREAD

**Issue by flosse** [DenisKolodin/mould#4]
Would you mind to add one or two examples?

**Comment by therustmonk** [comment]
Of course! API is still changing, but I will freeze it quite soon and add examples.
Thank you for your interest!

**Comment by flosse** [comment]
Great!

BTW: If you like, you could add your project to the [rust-web-framework-comparison](https://github.com/flosse/rust-web-framework-comparison)

**Comment by therustmonk** [comment]
Nice! Thanks!
